modules/quadbin/snowflake/Makefile

```diff
@@ -0,0 +1,4 @@
+MODULE = quadbin
+MODULE_DEPS = quadkey
```

Comment:
I'm not sure that if need this `MODULE_DEPS`. Normally we only add them in advanced not in core. I see that you are using `QUADINT_TOZXY` but normally the script modulesort.js will take care of sorting the deployment of all the modules.

This kind of topics I think will be nice to polish in Q3 so everything is detected and deployed automatically.